integrations/adding-an-integration/connecting-to-conda.md

```diff
@@ -0,0 +1,9 @@
+# Connecting to conda
+
+Connecting to conda requires you to have both [conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) and [conda-build](https://docs.conda.io/projects/conda-build/en/stable/install-conda-build.html) installed in your Aqueduct server's environment. Once installed, you can simply click the conda icon in your [integrations page](../../integrations.md) to connect.
+
+Once connected, we will set up a few base conda environments that allows you to run [operators](../../operators.md) with different dependencies. This set up process generally takes a few minutes. You can click into the connected conda integration to view the setup progress.
```

Comment:
We should either say why we set up multiple (because it's not obvious) or leave that detail out. 

It might be nice to include a screenshot of what the setup progress looks like?